site_scons/prereq_tools/base.py

```diff
@@ -262,11 +264,15 @@ def initialize(self, env):
         self.env = env
         self.__dry_run = env.GetOption('no_exec')
 
-    def run_commands(self, commands, subdir=None):
+    def run_commands(self, commands, subdir=None, env=None):
         """Runs a set of commands in specified directory"""
         if not self.env:
             raise Exception("PreReqComponent not initialized")
         retval = True
+        if env is not None:
+            passed_env = env
+        else:
+            passed_env = self.env
```

Comment:
FYI, this works in python
```suggestion
        passed_env = env or self.env
```

Reply:
It stopped working when I did this, not sure why.